fivetran_provider_async/triggers/fivetran.py

```diff
@@ -28,7 +28,7 @@ def __init__(
         polling_period_seconds: float,
         connector_id: str,
         fivetran_conn_id: str,
-        previous_completed_at: pendulum.DateTime,
+        previous_completed_at: pendulum.DateTime = None,
```

Comment:
```suggestion
        previous_completed_at: pendulum.DateTime | None = None,
```

Reply:
Done in [a430049](https://github.com/astronomer/airflow-provider-fivetran-async/pull/8/commits/a4300496e12aeba5d6673638067638d8ecdee9cc)